templates/form/textInput.tpl

```diff
@@ -43,7 +43,7 @@
 					{if $FBV_readonly} readonly="readonly"{/if}
 					value="{$FBV_value[$thisFormLocale]|escape}"
 					name="{$FBV_name|escape}[{$thisFormLocale|escape}]"
-					id="{$FBV_id|escape}-{$thisFormLocale|escape}{$uniqId}"
+					id="{$FBV_id|escape}-{$thisFormLocale|escape|escape:jqselector}{$uniqId}"
```

Comment:
I think the escaping requirements are the same for all use cases of `jqselector`, so it would just be `{$someId|escape:jqselector}`, and `jqselector` would take care of the rest.